lib/api/contacts.js

```diff
@@ -3,3 +3,71 @@ export const queryEmails = (data) => ({
     method: 'get',
     data
 });
+
+export const getContacts = ({ Page = 0, PageSize = 1000, LabelID = '' }) => ({
+    url: `contacts/?${PageID},${PageSize},${LabelID}`,
+    method: 'get',
+});
+
+export const getContactExport = ({ Page = 0, PageSize = 1000, LabelID = '' }) => ({
+    url: `contacts/export${PageID},${PageSize},${LabelID}`,
+    method: 'get',
+    data: { Code, Contacts, Total },
+});
+
+export const getContact = ({ contactID }) => ({
```

Comment:
Pass `contactID` directly.
```suggestion
export const getContact = (contactID) => ({
```

Reply:
Indeed...